lib/tasks/data_import/services/output_service_delimited_fields.rake

```diff
@@ -0,0 +1,55 @@
+namespace :services do
+    desc 'Output a list of all the data contained in the delimited fields'
+    task :display_delimited_field_values => :environment do
+      file_path = Rails.root.join('lib', 'seeds', 'tvp-services.csv')
```

Comment:
Sorry if this is a silly question 😄 Is this rake task a kind of verification step to see what the data will look like once it's imported, like a dry run?

Reply:
yeh - a sanity check, I've done a similar function in google but its a bit flakey. 

I can see this being useful later on as a kind of 'preview your import' page as well

Reply:
it also works as an additional form of validation!

Reply:
Cool, thanks! I've added a bit to the README about it.